### PR TITLE
Release 4.5.0

### PR DESCRIPTION
**Pull request by blackbaud-svossen**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-release-4.5.0](https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-release-4.5.0)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.5.0 (2020-11-11)
+
+- Added a phone field field testing fixture. [#77](https://github.com/blackbaud/skyux-phone-field/pull/77)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Got an extra "field" in there. Otherwise, looks good :)

Comment (blackbaud-svossen) [reply CHANGELOG.md]: aww dammit

**Review by Blackbaud-DeniseCisneros — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-release-4.5.0](https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-release-4.5.0)

**Pull request merged**